geonode/harvesting/models.py

```diff
@@ -446,6 +446,25 @@ def delete(self, using=None, keep_parents=False):
         return super().delete(using, keep_parents)
 
 
+class HarvestableResourceResult(models.Model):
+    session = models.ForeignKey("AsynchronousHarvestingSession", on_delete=models.CASCADE, related_name="results")
+    resource = models.ForeignKey("HarvestableResource", on_delete=models.CASCADE, related_name="harvesting_results")
```

Comment:
Instead of creating a new result model, can't we just use the `ExecutionRequest` as we do for the importer? In the end, it does the same